config/environments/production.rb

```diff
@@ -81,21 +81,10 @@
   config.active_record.dump_schema_after_migration = false
 
   config.action_mailer.default_options = {
-    from: 'where2help <no-reply@staging-where2help.herokuapp.com>',
-    reply_to: 'where2help <where2helpinaustria@gmail.com>'
+    from: 'Where2Help <noreply@where2help.wien>',
+    reply_to: 'Where2Help <office@where2help.wien>'
   }
-  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.delivery_method = :sendmail
   config.action_mailer.default_url_options = { host: ENV['FQDN'] }
```

Comment:
`FQN` is Heroku's default environment variable for the app url without http or https prefix. You have that set the same in your production environment?